lib/active_fedora/model.rb

```diff
@@ -5,6 +5,45 @@ module ActiveFedora
   # This module mixes various methods into the including class,
   # much in the way ActiveRecord does.
   module Model
+    def self.best_class_from_uris(uris, default: nil)
```

Comment:
When you get out of WIP mode, it would be good to see some YARD docs for this method, even if it's just a one-line description + some `@examples`, like the method beneath it.